CONTRIBUTING.md

```diff
@@ -77,14 +77,19 @@ If everything looks good, they will merge the code and release a new version whi
 
 <!-- References -->
 
-[pull requests]: https://github.com/mineiros-io/terraform-google-iam-custom-role/pulls
-[pre-commit-file]: https://github.com/mineiros-io/terraform-google-iam-custom-role/blob/main/.pre-commit-config.yaml
+<!-- markdown-link-check-disable -->
```

Comment:
this doesn't seem to be the latest file https://raw.githubusercontent.com/mineiros-io/terraform-module-template/main/CONTRIBUTING.md?token=GHSAT0AAAAAABQWMVTUNC3RJKEPESRXRBOGYZAZ2NA